server/src/widget_directory_watcher.coffee

```diff
@@ -13,6 +14,15 @@ module.exports = (directoryPath) ->
 
   init = ->
     watcher.on 'change', (filePath, info) ->
+      # fsevents returns decomposed Unicode and fs returns precomposed Unicode
+      # so normalize() is used to convert "U\xCC\x88" to "\xC3\x9C"
+      filePath = filePath.normalize()
+
+      # checks if widget directory is a symbolic link and ensures the correct path
+      if fs.lstatSync(directoryPath).isSymbolicLink()
+        symbolicLinkPath = fs.readlinkSync(directoryPath)
+        filePath = directoryPath + filePath.slice(symbolicLinkPath.length)
```

Comment:
Would it be possible to move this logic one level up? What I mean is, instead of watching the symbolic link dir and then convert file paths on every event, maybe it is possible to just watch the actual dir to begin with